src/test/java/com/redhat/cloud/common/clowder/configsource/ConfigSourceTest.java

```diff
@@ -144,6 +144,13 @@ void testLogCw() {
 
     }
 
+    @Test
+    void testLogNone() {
```

Comment:
just a hint :) since JUnit 5 there is no need to start each test method with "test".